src/twilio/twilio_endpoint.bal

```diff
@@ -113,6 +113,26 @@ public type Client client object {
         return mapJsonToSmsResponse(jsonResponse);
     }
 
+    # Send WhatsApp message from the given account-sid.
+    # + fromNo - Mobile number which the WhatsApp message should be sent from
+    # + toNo - Mobile number which the WhatsApp message should be received to
+    # + message - Message body of the WhatsApp message
+    # + return - If success, returns a Programmable SMS response object, else returns error
```

Comment:
```suggestion
    # + return - If success, returns a programmable SMS response object, else returns an error
```

Reply:
Done. Instead of use of "programmable SMS response" used "WhatsAppResponse"